lib/ansible/modules/system/selinux.py

```diff
@@ -177,13 +208,13 @@ def main():
         if module.check_mode:
             module.exit_json(changed=True)
         # cannot change runtime policy
-        msgs.append('reboot to change the loaded policy')
+        msgs.append('Running SELinux policy changed from \'%s\' to \'%s\'' % (runtime_policy, policy))
         changed = True
 
     if policy != config_policy:
         if module.check_mode:
             module.exit_json(changed=True)
-        msgs.append('config policy changed from \'%s\' to \'%s\'' % (config_policy, policy))
+        msgs.append('Config SELinux policy changed from \'%s\' to \'%s\'' % (config_policy, policy))
```

Comment:
s/Config/Configured ?

Or maybe even something like:

``` python
msgs.append('SELinux policy configuration in "%s" changed from \'%s\' to \'%s\'' % (configfile, config_policy, policy))

```

I would also move adding that message unti after set_config_policy() in case it fails.

( If we had a way to add info or debug level messages to the return, I would suggest logging a 'About to change selinux policy config in $CONFIG_FILE from $CONFIG_POLICY to $POLICY'. But we dont have that, so dont worry about it (module.lod/module.debug log only on remote)